api/client/info.go

```diff
@@ -43,16 +44,18 @@ func (cli *DockerCli) CmdInfo(args ...string) error {
 
 	fmt.Fprintf(cli.out, "Plugins: \n")
 	fmt.Fprintf(cli.out, " Volume:")
-	for _, driver := range info.Plugins.Volume {
-		fmt.Fprintf(cli.out, " %s", driver)
-	}
+	fmt.Fprintf(cli.out, " %s", strings.Join(info.Plugins.Volume, " "))
 	fmt.Fprintf(cli.out, "\n")
 	fmt.Fprintf(cli.out, " Network:")
-	for _, driver := range info.Plugins.Network {
-		fmt.Fprintf(cli.out, " %s", driver)
-	}
+	fmt.Fprintf(cli.out, " %s", strings.Join(info.Plugins.Network, " "))
 	fmt.Fprintf(cli.out, "\n")
 
+	if len(info.Plugins.Authorization) != 0 {
+		fmt.Fprintf(cli.out, " Authorization:")
```

Comment:
@cpuguy83 because by default there is no authorization plugin at all, but there is default volume and network plugins :stuck_out_tongue:.